taskt/Core/Automation/Commands/Error Handling Commands/GetExceptionMessageCommand.cs

```diff
@@ -1,57 +1,60 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Windows.Forms;
-using taskt.UI.CustomControls;
-using taskt.UI.Forms;
 using System.Linq;
+using System.Windows.Forms;
 using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
 using taskt.Core.Utilities.CommonUtilities;
+using taskt.UI.CustomControls;
+using taskt.UI.Forms;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("Error Handling Commands")]
-    [Attributes.ClassAttributes.Description("This command allows you to store an exception error message in a variable.")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to save an exception error message")]
-    [Attributes.ClassAttributes.ImplementationDescription("")]
+    [Group("Error Handling Commands")]
+    [Description("This command allows you to store an exception error message in a variable.")]
     public class GetExceptionMessageCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please define where the error text should be stored")]
-        [Attributes.PropertyAttributes.InputSpecification("Select or provide a variable from the variable list")]
-        [Attributes.PropertyAttributes.SampleUsage("**vSomeVariable**")]
-        [Attributes.PropertyAttributes.Remarks("If you have enabled the setting **Create Missing Variables at Runtime** then you are not required to pre-define your variables, however, it is highly recommended.")]
-        public string v_userVariableName { get; set; }
+        [PropertyDescription("Output String Variable")]
+        [InputSpecification("Choose a variable to take the output error string.")]
+        [SampleUsage("vSomeVariable")]
+        [Remarks("If you have enabled the setting **Create Missing Variables at Runtime** then you are not required " +
+                 "to pre-define your variables, however, it is highly recommended.")]
+        public string v_OutputUserVariableName { get; set; }
```

Comment:
Could you please follow the template in the guidelines? I'd like as much uniformity as possible for outputs, especially since we might later replace them all with a default tooltip. Change only the TYPE in bold. Maybe change String to Exception Message.

Reply:
Done